package.json

```diff
@@ -21,7 +21,7 @@
   "author": "Stephen Sawchuk <sawchuk@gmail.com>",
   "license": "MIT",
   "engines": {
-    "node": ">=4"
+    "node": ">=8.10.0"
```

Comment:
Shouldn't this just be `"8"`?

Reply:
I always forget why, but this matches what the Google API client libs use. 🤷‍♂